tests/devlxd-vm

```diff
@@ -87,7 +87,9 @@ lxc start v1
 waitVMAgent v1
 
 # Configure to use the proxy
-lxc exec v1 -- snap install lxd --channel="${LXD_SNAP_CHANNEL:-"latest/edge"}"
+lxc exec v1 -- snap wait system seed.loaded || true
```

Comment:
why `|| true` here?

Reply:
I'm afraid I don't remember, I'll drop it.